addons/ModelViewer3D/documentation.md

```diff
@@ -16,6 +16,10 @@ The list starts with the common properties, learn more about them by visiting th
         <td>Model</td>
         <td>Path to the file containing the 3D model. Supports only the glTF/GLB 3D models.</td>
     </tr>
+    <tr>
+        <td>Model iOS</td>
+        <td>Path to the file containing the 3D model necessary to run the AR functionality in iOS. Supports only the usdz 3D models.</td>
+    </tr>
```

Comment:
i tutaj wspomnieć. 
Check chapter `Augmented Reality (AR) support` for more information.